index.Rmd

```diff
@@ -64,7 +64,7 @@ race_eth_summarize <- race_eth_lgst %>%
   rename(c("Race/Ethnicity" = variable, "Number of Tracts" = n))
 
 kable(race_eth_summarize) %>%
-  kable_styling(bootstrap_options = "striped", full_width = F, position = "left")
+  kable_styling(bootstrap_options = "striped", full_width = F, position = "left") 
```

Comment:
I'm not sure if it is better that try to use the position = "center".

Reply:
Ok I will try this, thanks!